src/flask_odoo/__init__.py

```diff
@@ -46,8 +48,12 @@ def teardown(self, exception):
 
     def create_common_proxy(self):
         url = current_app.config["ODOO_URL"]
-        common = xmlrpc.client.ServerProxy(f"{url}/xmlrpc/2/common")
-        return common
+        if current_app.config["USE_UNVERIFIED_SSL_CONTEXT"].lower() == "true":
```

Comment:
another way to have a boolean setting is:

```
import ast

use_unverified_ssl_context = ast.literal_eval(current_app.config["USE_UNVERIFIED_SSL_CONTEXT"])
```

the you can set `current_app.config["USE_UNVERIFIED_SSL_CONTEXT"]` to `"True"` or `"False"`